app/assets/scripts/components/filter.js

```diff
@@ -156,108 +167,117 @@ export default function Filter({
+                <Dropdown
+                  triggerElement="a"
+                  triggerTitle="View country options"
+                  triggerText="Country"
+                  triggerClassName="button--drop-filter filter--drop"
                 >
-                  {_.sortBy(countries).map(o => {
-                    return (
-                      <li key={o.code}>
-                        <div
-                          data-cy="filter-menu-item"
-                          className={c('drop__menu-item', {
-                            'drop__menu-item--active': selected.countries.includes(
-                              o.code
-                            ),
-                          })}
-                          data-hook="dropdown:close"
-                          onClick={() => {
-                            onFilterSelect('countries', o.code);
-                          }}
-                        >
-                          <span data-cy={o.name}>{o.name}</span>
-                        </div>
-                      </li>
-                    );
-                  })}
-                </ul>
-              </Dropdown>
+                  <ul
+                    role="menu"
+                    data-cy="filter-countries"
+                    className="drop__menu drop__menu--select scrollable"
+                  >
+                    {_.sortBy(countries).map(o => {
+                      return (
+                        <li key={o.code}>
+                          <div
+                            data-cy="filter-menu-item"
+                            className={c('drop__menu-item', {
+                              'drop__menu-item--active': selected.countries.includes(
+                                o.code
+                              ),
+                            })}
+                            data-hook="dropdown:close"
+                            onClick={() => {
+                              onFilterSelect('countries', o.code);
+                            }}
+                          >
+                            <span data-cy={o.name}>{o.name}</span>
+                          </div>
+                        </li>
+                      );
+                    })}
+                  </ul>
+                </Dropdown>
+              )}
 
-              <Dropdown
-                triggerElement="a"
-                triggerTitle="View filter options"
-                triggerText="Parameter"
-                triggerClassName="button--drop-filter filter--drop"
-              >
-                <ParamSelect
-                  parameters={parameters}
-                  onFilterSelect={onFilterSelect}
-                  selected={selected}
-                />
-              </Dropdown>
+              {parameters && parameters.length > 1 && by.includes('countries') && (
+                <Dropdown
+                  triggerElement="a"
+                  triggerTitle="View filter options"
```

Comment:
Tests want this to say `View parameter options`, not sure why this wasn't failing previously